lib/ProkkaAnnotation/Util/ProkkaUtils.py

```diff
@@ -17,6 +17,7 @@
 from DataFileUtil.DataFileUtilClient import DataFileUtil
 from GenomeAnnotationAPI.GenomeAnnotationAPIClient import GenomeAnnotationAPI
 from GenomeFileUtil.GenomeFileUtilClient import GenomeFileUtil
+from installed_clients.kb_SetUtilitiesClient import kb_SetUtilities
```

Comment:
It would be a good idea to reinstall the GenomeAnnotationAPI, GenomeFileUtil, KBaseReportClient, and WorkspaceClient using the latest sdk

Reply:
Actually it looks like james other pr fixes that, so never mind.